package.json

```diff
@@ -1,16 +1,24 @@
 {
   "name": "swl-builder",
+  "homepage": "https://lukezors.github.io/swl-builder",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.15.1",
+    "gh-pages": "^2.1.1",
+    "prop-types": "^15.7.2",
     "react": "^16.11.0",
```

Comment:
You should be able to update React here without issues (after this PR if you want to wait) - you can run `npm outdated` to see what your versions are, `npm update` to update all minor versions, and `npm install <package>@<version>` to upgrade a major version!